test/cypress/e2e/search.spec.js

```diff
@@ -213,15 +211,4 @@ describe('Search', () => {
     cy.get(ss.searchInput).click().type('43th3j4bt324');
     cy.get(ss.searchNoResultMessage).eq(0).should('have.text', 'No results found');
   });
-
-
-  /**
-   * Search for nonexistent item
-   * @expect no results plug
-   */
-  it('Search for nonexistent item - shows no results plug', () => {
```

Comment:
Had to remove this test, since the string `43th3j4bt324` will be considered as a possible delegate name, and the enter key press will be delayed, so the user won't be sent to the `No results` page.

Reply:
You can put a made-up tx id instead